db/seeds.rb

```diff
@@ -5,3 +5,40 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+
+user = User.where(email: 'test@example.com').first_or_create(password: '12345678', password_confirmation: '12345678')
+
+apartments_array = [
+    {
+    street: "123 Snoring Ln.",
+    city: "New Foundland",
+    state: "ND",
+    manager: "Peter Petigrew",
+    email: "peter67@yahoo.com", 
+    price: "$250,000", 
+    bedrooms: 1, 
+    bathrooms: 1, 
+    pets: "No",
+    image: "https://thiscatdoesnotexist.com/"
+    },
+    {
+    street: "8750 Poland Dr.",
+    city: "Philadelphia",
+    state: "PA",
+    manager: "Ronald McDonald",
+    email: "rm1955@aol.com", 
+    price: "$740,000", 
```

Comment:
I feel sad that I have been conditioned to think that $740,000 is reasonable

Reply:
Still outside our price range, but maybe after this bootcamp that won't be the case 👀